src/m_menu.c

```diff
@@ -3996,6 +4007,10 @@ static const char *death_use_action_strings[] = {
   "default", "last save", "nothing", NULL
 };
 
+static const char *menu_background_strings[] = {
+  "on", "off", "dark tint", NULL
```

Comment:
Does "dim" fit better than "dark tint"? What do the native speakers say?

Reply:
"Dim" is better than "dark tint" for me, but I do not have a sense of how widespread that word is for non-native speakers. A simple "dark" might also get the job done here.